codegen/test_data/endpoints/bar_bar_method_toomanyargs.gogen

```diff
@@ -215,17 +225,19 @@ func convertToTooManyArgsClientRequest(in *endpointsBarBar.Bar_TooManyArgs_Args)
 		for key1, value2 := range in.Foo.FooMap {
 			out.Foo.FooMap[key1] = string(value2)
 		}
-		if in.Foo.Message != nil {
-			out.Foo.Message = &clientsFooBaseBase.Message{}
-			out.Foo.Message.Body = string(in.Foo.Message.Body)
-		} else {
-			out.Foo.Message = nil
-		}
+		convertToTooManyArgsMessageClientRequest(in, out)
```

Comment:
this method name contains the local field name instead of the absolute field name.

Should probably be `convertToTooManyArgsFooMessageClientRequest(in, out)`

If its a local message name then `{ foo: { message: 'a' }, bar: { message: 'b' } }` would cause duplicate method name bugs.